ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -212,11 +212,11 @@ public struct Avatar: View, TokenizedControlView {
                 Image(uiImage: image)
                     .renderingMode(avatarImageInfo.renderingMode)
                     .resizable()
-                    .foregroundColor(Color(dynamicColor: foregroundColor))
+                    .foregroundColor(Color(foregroundColor))
             } else {
                 Text(initialsString)
-                    .foregroundColor(Color(dynamicColor: foregroundColor))
-                    .font(.fluent(tokenSet[.textFont].fontInfo, shouldScale: false))
+                    .foregroundColor(Color(foregroundColor))
+                    .font(.init(tokenSet[.textFont].font)) // shouldScale: false
```

Comment:
Nit: is this comment deliberate?

Reply:
Ah yeah that's the comment I made earlier: it's a known WIP that I need to resolve before this can merge. Sorry for the confusion!

Reply:
Okay, we have real scaling again!